lib/ObjWriter/.nuget/Microsoft.DotNet.ObjectWriter.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Microsoft.DotNet.ObjectWriter</id>
-    <version>1.0.4-prerelease-00001</version>
+    <version>1.0.6-prerelease-00001</version>
```

Comment:
Can you change it to "1.0.5-prerelease-00002" instead? This is minor change that doesn't break the existing API. By increasing the postfix number, we don't need to update CoreRT consumption.
The same applies to all below.